packages/tuono/src/config/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TuonoConfig {
+  vite?: {
+    alias?: Record<string, string>
```

Comment:
Since we are passing this option directly to `vite` we might consider to extract this types from the `vite` config type itself rather than hardcoding them on our side